Sources/Purchasing/Configuration.swift

```diff
@@ -113,11 +113,19 @@ import Foundation
          *
          * - Important: Set this property if you have your own user identifiers that you manage.
          */
+        @_disfavoredOverload
         @objc public func with(appUserID: String?) -> Builder {
             self.appUserID = appUserID
             return self
         }
 
+        // swiftlint:disable:next missing_docs
+        public func with(appUserID: StaticString) -> Configuration.Builder {
+            Logger.warn(Strings.identity.logging_in_with_static_string)
+            self.appUserID = "\(appUserID)"
```

Comment:
I'd delegate this to the other method:

```suggestion
            return self.with(appUserID: "\(appUserID)")
```